lawlibrary/settings.py

```diff
@@ -3,7 +3,6 @@
 INSTALLED_APPS = [
     "lawlibrary.apps.LawlibraryConfig",
     "peachjam_subs",
-    "peachjam_ml",
```

Comment:
I had this as opt-in because it requires the database itself to have specific plugins enabled. We currently only have two websites that support this functionality.

Reply:
should we then also have the view itself live in this app? tests will fail otherwise

Reply:
yes, I think so for now. I'm not sure how complex that will get as we integrate similarity into other features, but let's start like that for now. It's not too complex to move later if necessary.

Reply:
moved to peachjam ml